src/plugins/direct_access_links/public/index.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { PluginInitializerContext } from '../../../core/public';
+import { DirectAccessLinksPlugin } from './plugin';
+
+export function plugin(initializerContext: PluginInitializerContext) {
+  return new DirectAccessLinksPlugin(initializerContext);
+}
+
+export { DirectAccessLinksSetup, DirectAccessLinksStart } from './plugin';
+
+export {
+  GeneratorId,
+  GeneratorState,
+  GeneratorStateMapping,
```

Comment:
What do you think about prefixing these with `DirectAccessLink*`?

Reply:
Here is some AppArch internal discussion: https://github.com/elastic/kibana/issues/52374#issuecomment-584197758

Reply:
Yea, makes sense. Whyyyy did I give this such a long name, haha. Maybe `access_link` is better, drop the `direct`. 🤔

Reply:
Or maybe prefix with `DAL*`.

Reply:
It could be just `links` plugin.